guides/gatherers.md

```diff
@@ -750,6 +751,43 @@ Example output (in Rhai):
 
 For extra information refer to [trento-project/agent/../gatherers/sbddump_test.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/sbddump_test.go)
 
+### syssysctltemd
```

Comment:
did you mean `sysctl`?

Reply:
I meant `sysctl`, but my fingers apparently disagree :sweat_smile:  thanks!